core/src/test/java/cz/xtf/core/openshift/ClusterVersionOpenShiftBinaryPathResolverTest.java

```diff
@@ -6,34 +6,61 @@
 
 import org.apache.commons.codec.digest.DigestUtils;
 import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.Test;
 
 import cz.xtf.core.config.OpenShiftConfig;
+import cz.xtf.core.config.XTFConfig;
 
 public class ClusterVersionOpenShiftBinaryPathResolverTest {
 
     private OpenShiftBinaryPathResolver resolver = new ClusterVersionOpenShiftBinaryPathResolver();
 
+    @AfterEach
+    public void cleanVersion() {
+        System.clearProperty(OpenShiftConfig.OPENSHIFT_VERSION);
```

Comment:
system-stubs library used in another module might be also used to deal with system properties

https://github.com/xtf-cz/xtf/blob/master/junit5/src/test/java/cz/xtf/junit5/extensions/SkipForConditionTest.java#L25

Reply:
added dependency and replaced with `@SystemStub`